man/man8/zdb.8

```diff
@@ -137,6 +137,11 @@ also display the configuration that would be used were the pool to be imported.
 Display information about datasets.
 Specified once, displays basic dataset information: ID, create transaction,
 size, and object count.
+See
+.Fl N
+for determining if [poolname[/dataset | objset ID]] is to use the specified
```

Comment:
Is this mimicking the SYNOPSIS lines? If so – use the right macros here.

Reply:
I was trying to mimic other places where zdb.8 says "see -<option>".  Is there a better syntax?
```
To specify the cache file to display, see
.Fl U .
.Pp
```

Reply:
No, that's fine, I mean the `[poolname[/dataset |` thing

Reply:
Fixed.